### PR TITLE
Add `_Concurrency` module.

### PR DESCRIPTION
**Pull request by zoecarver**
This is an empty module that silences compiler errors.

### CODE REVIEW

File: Sources/_Concurrency/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#[[
+Copyright © 2018 Saleem Abdulrasool <compnerd@compnerd.org>.
+All Rights Reserved.
+
+SPDX-License-Identifier: BSD-3
+#]]
+
+add_library(swiftConcurrencyShims
```
Comment (compnerd) [inline Sources/_Concurrency/CMakeLists.txt:8]: This needs to be `swift_Concurrency`.  The name of the library is used for the output, and I believe that the library name is expected to be `swift_Concurrency.dll`/`libswift_Concurrency.so`/`libswift_Concurrency.dylib`.
```suggestion
add_library(swift_Concurrency
```

File: Sources/_Concurrency/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  ConcurrencyShims.swift)
+set_target_properties(swiftConcurrencyShims PROPERTIES
```
Comment (compnerd) [inline Sources/_Concurrency/CMakeLists.txt:10]:
```suggestion
set_target_properties(swift_Concurrency PROPERTIES
```

File: Sources/_Concurrency/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  Swift_MODULE_NAME _Concurrency)
+target_compile_options(swiftConcurrencyShims PRIVATE
```
Comment (compnerd) [inline Sources/_Concurrency/CMakeLists.txt:12]:
```suggestion
target_compile_options(swift_Concurrency PRIVATE
```

File: Sources/_Concurrency/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  -parse-stdlib
+  -disallow-use-new-driver
+  "SHELL:-Xfrontend -enable-resilience")
+target_link_libraries(swiftConcurrencyShims PUBLIC
```
Comment (compnerd) [inline Sources/_Concurrency/CMakeLists.txt:16]:
```suggestion
target_link_libraries(swift_Concurrency PUBLIC
```

File: Sources/_Concurrency/CMakeLists.txt [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  swiftCore)
+target_link_options(swiftConcurrencyShims PRIVATE
+  "SHELL:-Xclang-linker -nostdlib")
```
Comment (compnerd) [inline Sources/_Concurrency/CMakeLists.txt:19]: I wonder if this is something that we should keep.  We do want the dependency on other language runtimes once the code will be added.

**Review by compnerd — CHANGES_REQUESTED** [review]
I'd rather have the empty file be named `_Concurrency.swift`.

File: Sources/_Concurrency/CMakeLists.txt (hunk 5 above)
Comment (zoecarver) [reply Sources/_Concurrency/CMakeLists.txt:19]: I don't feel strongly, but it might be good future proofing. And it matches what we do everywhere else.

Comment (compnerd) [reply Sources/_Concurrency/CMakeLists.txt:19]: Fair enough, leaving this for now won't hurt anything.

**Review by compnerd — APPROVED** [review]

**Comment by zoecarver** [comment]
@compnerd thanks for landing both of these!

**Pull request merged**